ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.17.yaml

```diff
@@ -757,6 +757,8 @@ tests:
     cluster_profile: gcp-qe
     env:
       COMPUTE_NODE_TYPE: t2a-standard-4
+      COMPUTE_SERVICE_ACCOUNT: jiwei-worker-sa@openshift-qe.iam.gserviceaccount.com
```

Comment:
Suggest to apply the change to the following `gcp-ipi-xpn-mini-perm-byo-hosted-zone-arm-f60` job too.

Reply:
As we [discussed](https://redhat-internal.slack.com/archives/CH76YSYSC/p1727249454761929?thread_ts=1725420767.912529&cid=CH76YSYSC) in Slack, only update `gcp-ipi-xpn-mini-perm` tests, but not `gcp-ipi-xpn-mini-perm-byo-hosted-zone` tests. Thanks!